bundles/http/Citrix/AdobeSign/31fff7a5-ffa8-4019-9e2f-e477837f92de.js

```diff
@@ -0,0 +1,588 @@
+        method: "POST",
+        body: formData,
+        headers: {
+            "x-api-user": `userid:${actionParameters.userid}`
+        }
+    })
+    const json = await response.json()
+    if (!response.ok) {
+        throw new Error(JSON.stringify(json))
+    }
+    return json.transientDocumentId;
+}
+
+integration.define({
+    synchronizations: [
+        {
+            name: "fullSync",
+            fullSyncFunction: fullSync,
+            incrementalSyncFunction: incrementalSync
+        }
+    ],
+    integrationParameters: [
+        { name: "GroupId", type: "STRING", label: "AdobeSign GroupId", required: true },
+        { name: "AgreementCount", type: "STRING", label: "Agreements Per User", required: true , description: "No of Agreements to stored per user in cache table" }
+    ],
+    model: {
+        tables: [
+            {
+                name: "agreements",
+                columns: [
+                    { name: "id", type: "STRING", primaryKey: true, length: 255 },
+                    { name: "displayDate", type: "DATETIME" },
+                    { name: "esign", type: "BOOLEAN" },
+                    { name: "type", type: "STRING", length: 255 },
+                    { name: "name", type: "STRING", length: 255 },
+                    { name: "groupId", type: "STRING", length: 255 },
+                    { name: "userId", type: "STRING", length: 255, primaryKey: true },
+                    { name: "latestVersionId", type: "STRING", length: 255 },
+                    { name: "status", type: "STRING", length: 255 },
+                    { name: "hidden", type: "BOOLEAN" },
+                ]
+            },
+            {
+                name: "libraryDocuments",
+                columns: [
+                    { name: "id", type: "STRING", primaryKey: true, length: 255 },
+                    { name: "name", type: "STRING", length: 255 },
+                    { name: "creatorEmail", type: "STRING", length: 255 },
+                    { name: "ownerEmail", type: "STRING", length: 255 },
+                    { name: "modifiedDate", type: "DATETIME" },
+                    { name: "sharingMode", type: "STRING", length: 255 },
+                    { name: "groupId", type: "STRING", length: 255 },
+                    { name: "status", type: "STRING", length: 255 },
+                    { name: "hidden", type: "BOOLEAN" },
+                ]
+            },
+            {
+                name: "templateTypes",
+                columns: [
+                    { name: "templateType", type: "STRING", length: 255 },
+                    { name: "parentId", type: "STRING", length: 255 },
+                    { name: "uniqueId", type: "STRING", length: 255, primaryKey: true }
+                ]
+            },
+            {
+                name: "Users",
+                columns: [
+                    { name: "ID", type: "STRING", primaryKey: true, length: 255 },
```

Comment:
I don't understand "the logic" behind field naming.
* here `"ID"` - all letters uppercase vs. line 422 - `"id"`- all letters lowercase
* line 464 "userid" vs line 413 `"userId"`
* line 472 `"agreement_id"` vs line 484 `"libraryDocumentId"` - with `_` and without
* sometimes used just `"id"` sometimes full name `"libraryDocumentId"`
* table name line 434 vs line 442 - first uppercase, second lowercase
* same naming relationships, actions,...

=> very confusing

Reply:
Here, now we cannot modify the primary key for the table. for non-primary key column i'll do that changes

Reply:
Why not?

Reply:
Changes implemented for ID -> id and participent_id -> participant_id.